packages/bpk-docs/src/pages/InfiniteScrollPage/InfiniteScrollPage.js

```diff
@@ -0,0 +1,202 @@
+    ))}
+  </BpkList>
+);
+
+List.propTypes = {
+  elements: PropTypes.arrayOf(PropTypes.any).isRequired,
+};
+
+const FixedHeightPanel = ({ children }) => (
+  <div className={getClassName('bpk-docs-infinite-scroll-page__fixed-panel')}>
+    {children}
+  </div>
+);
+
+FixedHeightPanel.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+const CustomSeeMore = ({ onSeeMoreClick }) => (
+  <div
+    className={getClassName('bpk-docs-infinite-scroll-page__custom-component')}
+  >
+    <BpkButton onClick={onSeeMoreClick}>See More</BpkButton>
+  </div>
+);
+
+CustomSeeMore.propTypes = {
+  onSeeMoreClick: PropTypes.func.isRequired,
+};
+
+const CustomLoading = () => (
+  <div
+    className={getClassName('bpk-docs-infinite-scroll-page__custom-component')}
+  >
+    <BpkSpinner type={SPINNER_TYPES.primary} />
+  </div>
+);
+
+const InfiniteList = withInfiniteScroll(List);
+const defaultFetch = onItemsFetch(elementsArray);
+
+const components = [
+  {
+    id: 'default',
+    title: 'Default',
+    blurb: 'Loads five more list items upon reaching the bottom of the list.',
+    examples: [
+      <FixedHeightPanel>
+        <InfiniteList
+          onItemsFetch={(index, nElements) =>
+            new Promise(resolve => {
+              setTimeout(() => {
+                const elements = [];
+                for (let i = 0; i < nElements; i += 1) {
+                  elements.push(`Item ${index + i}`);
+                }
+                resolve(elements);
+              }, 10);
+            })
+          }
+        />
+      </FixedHeightPanel>,
+    ],
+  },
+  {
+    id: 'one-element-per-scroll',
+    title: 'One element per scroll',
+    blurb: 'Loads one more list item upon reaching the bottom of the list.',
+    examples: [
+      <FixedHeightPanel>
+        <InfiniteList onItemsFetch={defaultFetch} elementsPerScroll={1} />
+      </FixedHeightPanel>,
+    ],
+  },
+  {
+    id: 'partial-load',
+    title: 'Partial load',
+    blurb:
+      'After 15 items have loaded, a button must be pressed to load more items.',
+    examples: [
+      <FixedHeightPanel>
+        <InfiniteList
+          onItemsFetch={defaultFetch}
+          elementsPerScroll={5}
+          seeMoreAfter={2}
+          renderSeeMoreComponent={CustomSeeMore}
+        />
+      </FixedHeightPanel>,
+    ],
+  },
+  {
+    id: 'custom-loading',
+    title: 'Custom loading component',
+    blurb: [],
+    examples: [
+      <FixedHeightPanel>
+        <InfiniteList
+          onItemsFetch={(index, nElements) =>
+            new Promise(resolve => {
+              setTimeout(() => {
+                resolve(defaultFetch(index, nElements));
+              }, 500);
+            })
+          }
+          elementsPerScroll={5}
+          renderLoadingComponent={CustomLoading}
+        />
+      </FixedHeightPanel>,
+    ],
+  },
+];
+
+const blurb = [
+  <IntroBlurb>
+    This High-order component is used to wrap a list component and add the
```

Comment:
This should say `higher-order component`, not `High-order component`.

Sorry - I meant to add this to my original review and forgot.